### ISSUE TITLE
Digital product upload file and sample file

### ISSUE THREAD

**Issue by rafesposo** [openstream/No-Flash-Image-Uploader#20]
It seems that in the digital product is still using Flash, can we upgrade so that it works there too?

**Comment by nickweisser** [comment]
Not sure what you mean by digital product. You're welcome you send pull requests our way ;-)

**Comment by neonmaccca** [comment]
I have this problem also.

This plugin works great for the most part, however, when you try to create a downloadable product and then upload Downloadable Information, the flash upload is still present.

When trying to add a new row within this area I get the following error to the console, 
`Uncaught ReferenceError: Flex is not defined
    at klass.initialize ((index):1733)
    at klass (prototype.js:101)
    at Object.add ((index):2255)
    at HTMLButtonElement.<anonymous> (prototype.js:391)
    at HTMLButtonElement.responder (prototype.js:5598)`

Is there any intention to include support for this as it looks like it breaks existing functionality?
Screenshot attached
![screen](https://user-images.githubusercontent.com/12829576/28620292-377c6672-7204-11e7-9ae1-a58fcf42d042.jpg)

**Comment by zuiko** [comment]
Yes some years after, I have the same issue, if there is no solution for the moment, a workaround could be to use "URL" instead of "File" for the downloadable product.

**Comment by davidsbaldwin** [comment]
I have the same problem... did anybody ever figure this out?
